modules/op-changing-source-branch-in-repository-crd.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="changing-source-branch-in-repository-crd_{context}"]
+= Changing the source branch for the pipeline definition in the `Repository` CRD
+
+[role="_abstract"]
+By default, when processing a `push` or `pull request` event, {pac} fetches the pipeline definition from the branch that triggered the event. You can use the `pipelinerun_provenance` setting in the `Repository` CRD to fetch the definition from the default branch configured on the Git repository provider, such as `main`, `master`, or `trunk`.
+
+[source,yaml]
+----
+...
+spec:
+  ...
+  settings:
+    pipelinerun_provenance: "default_branch"
+  ...
+----
```

Comment:
I have done so for the YAML examples in the PR and for *some* YAML examples in the existing modules of the assembly. I have not done so in the existing examples where this would not be an obvious change, as I do not have the time to circle back to Dev.